pontoon/base/utils.py

```diff
@@ -332,6 +333,7 @@ def require_AJAX(f):
     """
     AJAX request required decorator
     """
+    @functools.wraps(f)
```

Comment:
Awww... I wonder if there's a some kind of pre-caution we should take to avoid similar bugs in the future?